Framework/Kernel/inc/MantidKernel/NullValidator.h

```diff
@@ -1,11 +1,10 @@
 #ifndef MANTID_KERNEL_NULLVALIDATOR_H_
 #define MANTID_KERNEL_NULLVALIDATOR_H_
 
-//----------------------------------------------------------------------
-// Includes
-//----------------------------------------------------------------------
 #include "MantidKernel/IValidator.h"
 
+#include <boost/make_shared.hpp>
```

Comment:
Would it make sense to define the `NullValidator::clone()` method in the .cpp file instead?